pkg/kubelet/prober/prober.go

```diff
@@ -49,16 +49,11 @@ const maxProbeRetries = 3
 
 // Prober helps to check the liveness/readiness/startup of a container.
 type prober struct {
-	exec execprobe.Prober
-	// probe types needs different httpprobe instances so they don't
-	// share a connection pool which can cause collisions to the
-	// same host:port and transient failures. See #49740.
```

Comment:
n.b. https://github.com/kubernetes/kubernetes/issues/49740 was a workaround, fixed upstream in golang 1.10+ with https://github.com/golang/go/issues/22091

Should be safe to collapse these.